src/sentry/issues/endpoints/project_events.py

```diff
@@ -31,26 +38,42 @@ class ProjectEventsEndpoint(ProjectEndpoint):
         }
     }
 
+    @extend_schema(
+        operation_id="List a Project's Error Events",
+        parameters=[
+            GlobalParams.ORG_ID_OR_SLUG,
+            GlobalParams.PROJECT_ID_OR_SLUG,
+            CursorQueryParam,
+            OpenApiParameter(
+                name="full",
+                description="If this is set to true then the event payload will include the full event body, including the stacktrace. Set to 1 to enable.",
+                required=False,
+                type=bool,
+                location="query",
+                default=False,
+            ),
+            OpenApiParameter(
+                name="sample",
+                description="Return events in pseudo-random order. This is deterministic, same query will return the same events in the same order.",
+                required=False,
+                type=bool,
+                location="query",
+                default=False,
+            ),
+        ],
+        responses={
+            200: inline_sentry_response_serializer(
+                "ProjectEventsResponseDict", list[SimpleEventSerializerResponse]
```

Comment:
SimpleEventSerializerResponse has SimpleEventTag which has a param query. Now query looks like it's optional but that's not how documentation reads it so it thinks it's required. You should use totality to say it's actually optional.
Here's more explained version in docs: https://develop.sentry.dev/application/api/public/#success-responses
Ping us if you have questions.